static/index.js

```diff
@@ -4,7 +4,7 @@ let currentMediaArray = [];
 let currentMediaTypesArray = [];
 let selectedMedia = [];
 let selectionMode = false;
-let section = "img";
+let section = localStorage.getItem('default') || "img";
```

Comment:
change the key name from `default` to `defaultSection`

Reply:
Okay!

Reply:
nicely done